frontend_tests/node_tests/stream_data.js

```diff
@@ -605,3 +605,7 @@ zrequire('marked', 'third/marked/lib/marked');
     assert.equal(antarctica_sub.color, '#76ce90');
 }());
 
+(function test_invite_streams() {
```

Comment:
I'd add a little comment here explaining how the data got set up, or, better yet, put the assertion closer in the module to where the data gets set up.  (It's ok in the node tests to test a bunch of really simple methods like this in a single function.  If you do split out the single function, it's often good to re-initialized the data and do specific setup, including a stream that's a "red herring" to make sure the real code properly ignores uninvited streams.

Reply:
I would add comment here since the data isn't setup anywhere, current we add data by calling `initialize` function that sets property to `page_params` which are delete or garbage collected by the function at the end once it sets up the default i assume.
cc @lonerz.